vpn/src/main/java/xyz/hexene/localvpn/ByteBufferPool.java

```diff
@@ -36,9 +39,22 @@ public static ByteBuffer acquire() {
     public static void release(ByteBuffer buffer) {
         buffer.clear();
         pool.offer(buffer);
+        atomicUpdateAndGet(allocations, value -> value > 0 ? value - 1 : 0);
```

Comment:
should this be part of this PR?

Reply:
_Just clarified offline that this is needed to record the allocations_